engine/buildkit/progrock.go

```diff
@@ -169,6 +162,19 @@ func BK2Progrock(event *bkclient.SolveStatus) *progrock.StatusUpdate {
 				vtx.Error = &msg
 			}
 		}
+
+		g.recordsMu.Lock()
+		if op, ok := g.records[v.Digest]; ok {
+			if op != nil {
+				g.records[v.Digest] = nil // don't write out a record again
+
+				if a, err := types.MarshalAny(op); err == nil {
+					status.Metas = append(status.Metas, &progrock.VertexMeta{Vertex: vtx.Id, Data: &anypb.Any{TypeUrl: a.TypeUrl, Value: a.Value}})
```

Comment:
Just need a `Name` here to distinguish it from any other meta attributes:

```suggestion
					status.Metas = append(status.Metas, &progrock.VertexMeta{Name: "op", Vertex: vtx.Id, Data: &anypb.Any{TypeUrl: a.TypeUrl, Value: a.Value}})
```